### ISSUE TITLE
Move mobile number field to core table

### ISSUE THREAD

**Issue by voltan** [pi-engine/pi#1580]
At this moment we use mobile number as extra field on user module. we can move it to core user account table active login by mobile number too , I think extra filed for valid mobile is required. what is your idea?

**Comment by Marc-pi** [comment]
well, i've checked, all fields are set in user/config/user.php by default (you can overwritte through (/usr/custom/module/user/config)
so there is already a field in DB : see `user_field` table
in that list, there is the phone and mobile phone number

you want to move solely this field to core_user_account ??
we will have issues, because user fields are used everywhere and in order module too.

login with mobile number : with SMS validation (as 2nd factor check) ??
with no SMS 2nd factor check, i doubt it is usefull. What is your need ?

**Comment by voltan** [comment]
Yes, at this moment this field exist , than we just move it to core_user_account, it will not  make any problems on any other modules ( like order ) because on all modules get full user information ( both of core and user module fields ) and after change is still will work 

Move mobile number to core, just help use to build user login by mobile ( at this moment I keep mobile number on username )
